index.bs

```diff
@@ -172,7 +174,49 @@ When asked to <dfn abstract-op>create or override the cached client hints set</d
  <li>[=map/Set=] [=Accept-CH cache=][|origin|] to |hintSet|.
 </ol>
 
-<dfn>Delegate-CH</dfn> {#delegate-ch-algo}
+`Critical-CH` {#critical-ch-algo}
+--------
+
+If an [=url/origin=] is loaded and the server sets an `Accept-CH` header that
+lists hints not already in the current [=Accept-CH cache=] that means only
+subsiquent loads of that [=url/origin=] will include the hints. If it's
+critical that every load (including the first) has the requested Client Hints,
+then the server can set a `Critical-CH` header to request a reload. The
+`Critical-CH` header itself does not modify the [=Accept-CH cache=].
+
+A reload will only occur when a hint in the `Accept-CH` header is both *not in*
+the [=Accept-CH cache=] and *in* the `Critical-CH` header. If hints listed in
+the `Critical-CH` header are already in the [=Accept-CH cache=] no reload is
+needed as they were sent. If hints listed in the `Critical-CH` header are not
+in the `Accept-CH` header a reload would not result in the hints being included
+anyway so none should be done to prevent an infinite loop.
+
+When asked if the user agent <dfn abstract-op>should reload page for critical client hints</dfn> given a |settingsObject| and |response|:
```

Comment:
Does this match https://datatracker.ietf.org/doc/html/draft-davidben-http-client-hint-reliability#section-3 (I haven't read closely yet - just posting here before I go on break)?

Reply:
It's missing the detection of a prior reload (preventing infinite reloads), but I'm not quite sure how to integrate that. Might need to leave it as a TODO for now